en/docs/learn/user-account-suspension.md

```diff
@@ -5,10 +5,10 @@ accounts that have been idle for a pre-configured amount of time.
 
 Prior to account suspension, set up the notification system to send a
 warning notification to the user announcing that the account will be
-suspended. For instance, if a user has not logged in to his/her account
-for 90 days, the user can be notified that his account will be suspended
+suspended. For instance, if users have not logged in to their accounts
```

Comment:
...if a user has not logged in to their account for 90 days, the user can be notified that the account will be suspended...

Reply:
clearer if we keep it singular IMO